<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-width: 2px;
	border-style: solid;
	border-color:gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="train_image_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border-width: 2px;
	border-style: solid;
	border-color:gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>379</width>
       <height>259</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_window_button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Train Image</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>171</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>max number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="threshold_line"/>
       </item>
       <item>
        <widget class="QLineEdit" name="numbermax_line"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
